src/components/organisms/AssetActions/Compute/FormComputeDataset.tsx

```diff
@@ -125,7 +126,16 @@ export default function FormStartCompute({
         algorithmDDO,
         accountId.toLowerCase()
       )
-      if (consumable) setAlgorithmConsumableStatus(consumable.status)
+      if (!consumable?.result) {
```

Comment:
question(blocking): I believe `consumable.result` is a boolean value, right? In that case:
issue/nitpick(blocking):  this should probably be check for `false`, rather than a falsy value, to avoid  entering this with `undefined`  for `consumable.result`

```tsx
if(consumable?.result === false) ...
```